capstone-app/src/app/modules/photos/components/image-container/image-container.component.scss

```diff
@@ -24,7 +23,7 @@
 
     .details {
       span {
-        bottom: 0;
+        bottom: -10px;
```

Comment:
Shouldn't this be top: 10px? Is this for absolute or relative positioning somewhere? I don't think this should be used, especially for span elements.

Reply:
This styling is for the photo titles and and the purpose is a transition where the caption 'swings up'. I know intuitively, changing bottom -10 to top 10 should do the same, but it doesnt in this case. I have three options: either switch the bottoms to the equivlent top pixels which would require new media queries for each size as it's a function of the height of the image, change it back to 0 px, or leave it. Do you have any preference?